rust/crd/src/authentication.rs

```diff
@@ -126,8 +126,7 @@ impl AirflowAuthentication {
                 // make sense to iterate further after finding an unsupported AuthenticationClass.
                 Some(match resolved.spec.provider {
                     AuthenticationClassProvider::Ldap(_) => resolved,
-                    AuthenticationClassProvider::Tls(_)
-                    | AuthenticationClassProvider::Static(_) => {
+                    _ => {
```

Comment:
It would be great if you could list the not supported providers explicitly.
The idea is to get a compile error for every new provider we need and you need to think about if this product supports the provider or not.

Reply:
Fixed in eda48f4763a061eaf2679fbb6c6482afd363b618